### ISSUE TITLE
WADebug requires a config file:

### ISSUE THREAD

**Issue by ahmedalmulki** [WhatsApp/WADebug#2]
I get the same msg whenever I want to run the wadebug command, I tried to create wadebug.conf.yml much time and configured correctly but without any luck, the same msg 
` file: wadebug.conf.yml in the current directory in order to run full checks, but none has been found. Do you want to create the file now? ` !!

**Comment by alanhhwong** [comment]
Have you seen the docs on this topic? https://developers.facebook.com/docs/whatsapp/guides/wadebug/#config

You will need to fill in values.  Can you share your config?

**Comment by ahmedalmulki** [comment]
yes, I already read that topic many times, and I filled all values as requested bu no luck.
`db:
  host:127.0.0.1
  port:3306
  user:username
  password:pass
webapp:
  baseUrl:https://ip:port
  user:admin
  password:pass
`

**Comment by alanhhwong** [comment]
can you share terminal prompts?
1) `pwd` and `ls`
2) `docker ps`
3) `wadebug`

**Comment by ahmedalmulki** [comment]
`root@WhatsAppPr:~/biz# ls
db.env  debug_output.txt  prod-docker-compose-old.yml  prod-docker-compose.yml  wadebug.conf.yml  wadebug_logs  wadebug_logs.zip
root@WhatsAppPr:~/biz# clear
root@WhatsAppPr:~/biz# ls
db.env  debug_output.txt  prod-docker-compose-old.yml  prod-docker-compose.yml  wadebug.conf.yml  wadebug_logs  wadebug_logs.zip
root@WhatsAppPr:~/biz# pwd
/root/biz
root@WhatsAppPr:~/biz# docker ps
CONTAINER ID        IMAGE                                 COMMAND                  CREATED             STATUS              PORTS                    NAMES
3b893550c8c7        docker.whatsapp.biz/web:v2.29.3       "/opt/whatsapp/bin/w…"   4 days ago          Up 4 days           0.0.0.0:11090->443/tcp   biz_waweb_1
153fa55f0bd8        docker.whatsapp.biz/coreapp:v2.29.3   "/opt/whatsapp/bin/w…"   4 days ago          Up 3 days           6250-6253/tcp            biz_wacore_1
root@WhatsAppPr:~/biz# wadebug

WADebug requires a config file: wadebug.conf.yml in the current directory in order to run full checks, but none has been found. Do you want to create the file now? [y/N]:
`

**Comment by destroyeramit** [comment]
I'm also facing same issue. Even all configuration parts are all set as per your docs.

![image](https://user-images.githubusercontent.com/32033464/94232175-0e797300-ff23-11ea-9f6f-73b6687d3fd9.png)

**Comment by tharkay** [comment]
I had the same problem and could narrow the problem down after editing the code in config.py [(line 51)](https://github.com/WhatsApp/WADebug/blob/894d86857ede7ec7007d19a57d6aaab02cba77fd/wadebug/config.py#L51):

    except Exception as e:
        raise e
        ....

This gave me the actual error (because the file was found but a different error was triggered):
> AttributeError: module 'yaml' has no attribute 'FullLoader'

[An answer on Stackoverflow](https://stackoverflow.com/a/63480207/410143) fixed this error message by simply upgrading my version of PyYAML. Use the flag `--ignore-installed` for upgrading, e.g. `pip install --ignore-installed PyYAML`

Wadebug is now finally working for me. Hopefully this might help someone else as well!
